src/components/ButtonsForFavourites/ButtonForFavourites.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { useAppSelector } from "../../hooks/useAppSelector";
+import { DeleteOutline, FavoriteBorder } from "@mui/icons-material";
+import { useAppDispatch } from "../../hooks/useAppDispatch";
+import {
+  addToFavourites,
+  deleteChekedMovie,
+} from "../../store/reducers/favouriteSlice";
+
+const ButtonForFavourites = ({ movie }) => {
+  const dispatch = useAppDispatch();
+  const { favouriteMovie } = useAppSelector(
+    (state) => state.movieFavouriteReducer
+  );
+  return (
+    <>
+      {!favouriteMovie.find((el) => el.imdbID === movie.imdbID) ? (
```

Comment:
favouriteMovie is an array. So maybe rename it with favouriteMovies.
And when u map the array cal element of array like favouriteMovie.